docs/operations/scheduler.md

```diff
@@ -0,0 +1,111 @@
+# KubeVirt Scheduler
+Scheduling is the process of matching Pods/VMs to Nodes. By default, the scheduler used is 
+[kube-scheduler](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-scheduler/).
+Further details can be found at [Kubernetes Scheduler Documentation](https://kubernetes.io/docs/concepts/scheduling-eviction/kube-scheduler/).
+
+## Creating a Custom Scheduler
+KubeVirt is compatible with custom schedulers. The configuration steps are described in the [Official Kubernetes 
+Documentation](https://kubernetes.io/docs/tasks/extend-kubernetes/configure-multiple-schedulers).
+Nevertheless, the Kubernetes version KubeVirt is running on and the Kubernetes version used to build the custom
```

Comment:
This makes more sense as a "Please note" than a "Nevertheless".

Reply:
Done.